### PR TITLE
Include 'is_active' in Device interface

### PR DESCRIPTION
**Pull request by dimitrisnl**
Connects-To: #516
Change-Type: minor
---- Autogenerated Waffleboard Connection: Connects to #516

### CODE REVIEW

**Comment by dimitrisnl** [comment]
Should be merged once v.5.1.0 gets in production

File: lib/models/device.coffee [hunk 1]
```diff
@@ -562,6 +562,36 @@ getDeviceModel = (deps, opts) ->
...
+	# });
+	#
+	# @example
+	# resin.models.device.isActive('7cf02a6', function(error, isActive) {
+	# 	if (error) throw error;
+	# 	console.log('Is device online?', isActive);
+	# });
+	###
+	exports.isActive = (uuidOrId, callback) ->
+		exports.get(uuidOrId, $select: 'is_active').get('is_active').asCallback(callback)
```
Comment (pimterry) [inline lib/models/device.coffee]: What's the use case for this method? We generally don't want to add a separate method in the SDK for each field, unless there's a really good reason somebody would need an easy way to specifically look up this one field.

In the UI for example, I imagine we'd load this as part of the `devices.getAll()/device.get(id)` call when loading the content for the page, not with a separate request for each device involved.

File: tests/integration/models/device.spec.coffee [hunk 2]
```diff
@@ -412,6 +412,24 @@ describe 'Device Model', ->
 				promise = resin.models.device.isOnline(999999)
 				m.chai.expect(promise).to.be.rejectedWith('Device not found: 999999')
 
+		describe.only 'resin.models.device.isActive()', ->
```
Comment (pimterry) [inline tests/integration/models/device.spec.coffee]: Don't commit `.only`! This disables the whole of the rest of the SDK test suite.

It's easy to miss ofc. Good opportunity for a shameless plug: https://github.com/pimterry/git-confirm (with `git config --add hooks.confirm.match ".only"`)

**Comment by dimitrisnl** [comment]
Oh, I assumed it was necessary to have one. Will remove it if it won't
serve any purpose.

**Comment by dimitrisnl** [comment]
Kept the bare minimum

**Pull request merged**